### PR TITLE
Use consistent compression and archive naming.

### PR DESCRIPTION
**Pull request by drewmoseley**
Signed-off-by: Drew Moseley <drew.moseley@northern.tech>

### CODE REVIEW

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -33,6 +33,6 @@ cp /work/uboot-mender/integration-binaries/*.tar* /output
 
 # Beagle Bone Black
 /usr/local/bin/build-uboot-bbb.sh
-cp /work/beaglebone_black*.tar /output
+cp /work/beaglebone_black*.tar* /output
```
Comment (lluiscampos) [inline docker-entrypoint.sh]: Why don't we use here explicit .tar.gz?

Comment (drewmoseley) [reply docker-entrypoint.sh]: For consistency with the other platforms.  They all use .tar*.  Let me know if you want me to change them all.

Comment (lluiscampos) [reply docker-entrypoint.sh]: Ah, ok. I guess they are leftovers from when we had the 00x prefixes. No strong opinion, let's leave it as-is.

**Comment by lluiscampos** [comment]
We will not merge this PR until we are very close to the release, see https://tracker.mender.io/browse/MEN-3202

**Pull request merged**